site/public/ts/controllers/MainDesignerController.ts

```diff
@@ -31,11 +31,13 @@ import {ANDGate}  from "../models/ioobjects/gates/ANDGate";
 import {ORGate}  from "../models/ioobjects/gates/ORGate";
 import {XORGate}  from "../models/ioobjects/gates/XORGate";
 import {LED}      from "../models/ioobjects/outputs/LED";
+import { SelectionPopupController } from "./SelectionPopupController";
```

Comment:
{SelectionPopupController}*

Reply:
With-spaces is the preferred syntax in the TS community. Still go ahead with the changes? If so, there are more instances of the same syntax in a few other files, not all from this PR. Should I change those too, or is it more about consistency per-file?